openlibrary/plugins/openlibrary/js/my-books/MyBooksDropper.js

```diff
@@ -86,7 +90,9 @@ export class MyBooksDropper extends Dropper {
 
         this.readingLogForms.initialize()
         this.readingLists.initialize()
-        this.checkInComponents.initialize()
+        if (this.checkInComponents) {
+            this.checkInComponents.initialize()
+        }
```

Comment:
This might be a good place for the JS optional chaining operator:
```suggestion
        this.checkInComponents?.initialize()
```